cypress/e2e/v1.0/pages/offence-analysis/questions/elementsOfTheOffence.ts

```diff
@@ -60,6 +60,42 @@ export default (stepUrl: string, summaryPage: string, positionNumber: number) =>
       })
     })
 
+    Array.of('Weapon').forEach(option => {
```

Comment:
The `forEach` seems redundant here... maybe a `const option = "Weapon"` on line 65 would be better 🤔